packages/babel-preset-cli/index.js

```diff
@@ -6,9 +6,13 @@ module.exports = () => ({
         targets: {
           node: '8.9.0',
         },
+        modules: false,
       },
     ],
     '@babel/preset-typescript',
   ],
-  plugins: ['@babel/plugin-proposal-class-properties'],
+  plugins: [
+    '@babel/plugin-proposal-class-properties',
+    ['@babel/plugin-transform-modules-commonjs', { lazy: source => true }],
```

Comment:
would just `lazy: true` work?

Reply:
According to [`@babel/plugin-transform-modules-commonjs` documentation](https://babeljs.io/docs/en/next/babel-plugin-transform-modules-commonjs.html#lazy), when set to `true` it will not make local imports lazy.